.github/workflows/email.yml

```diff
@@ -0,0 +1,83 @@
+name: email
+
+on:
+  push:
+    branches:
+      - main 
+      
+jobs:
+  
+  
+  send_email:
+    runs-on: ubuntu-latest
+  
+    steps:
+   
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
```

Comment:
Is there a need to change the default depth?

Reply:
Ah, probably for changed-files to get the diff. Can you add a comment saying something like that?

Reply:
Done